dsgrid/config/dataset_schema_handler_growth_rate.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+import logging
+
+from dsgrid.config.dataset_config import (
+    DatasetConfig,
+    check_load_data_filename,
+    check_load_data_lookup_filename,
+)
+from dsgrid.utils.spark import read_dataframe, get_unique_values
+from dsgrid.utils.timing import timer_stats_collector, Timer
+from dsgrid.config.dataset_schema_handler_base import DatasetSchemaHandlerBase
+from dsgrid.dimension.base_models import DimensionType
+from dsgrid.exceptions import DSGInvalidDataset, DSGInvalidDimension
+
+logger = logging.getLogger(__name__)
+
+
+class GrowthRateDatasetSchemaHandler(DatasetSchemaHandlerBase):
+    """ define interface/required behaviors for GROWTH_RATE dataset schema """
+
+    def __init__(self, config):
+        self._config = config
+
+    def check_consistency(self):
+        path = Path(self._config.model.path)
+        load_data_df = read_dataframe(check_load_data_filename(path), cache=True)
+        load_data_df = self._config.add_trivial_dimensions(load_data_df)
+        with Timer(timer_stats_collector, "check_one_table_data_consistency"):
+            self._check_one_table_data_consistency(self._config, load_data_df)
+        with Timer(timer_stats_collector, "check_metric_units"):
+            self._check_metric_units(self._config)
+
+    def _check_one_table_data_consistency(self, config: DatasetConfig, load_data):
```

Comment:
Is this the exact same code as OneTableDatasetSchemaHandler? If so, is there a way to share the code?